src/main/java/reposense/git/GitDiff.java

```diff
@@ -18,4 +21,18 @@ public static String diffCommit(String root, String lastCommitHash) {
         Path rootPath = Paths.get(root);
         return runCommand(rootPath, "git diff -U0 " + lastCommitHash);
     }
+
+    /**
+     * Returns the raw message of the number of lines modified of a given file {@code relativePath} form the commit
+     * {@code priorCommitHash} to the commit {@code lastCommitHash}.
+     */
+    public static String diffNumLinesModified(Path rootPath, String relativePath,
+            Optional<String> fromCommitHash, Optional<String> toCommitHash) {
+        String priorCommitHash = (fromCommitHash.isPresent()) ? fromCommitHash.get() : EMPTY_COMMIT_HASH;
```

Comment:
If `fromCommitHash` and `toCommitHash` are always `Optional.empty` when you call it, then it is unnecessary to have these two arguments.